keps/sig-node/2040-cri-beta/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Promote CRI API to beta
+kep-number: 2040
+authors:
+  - "@mrunalp"
+  - "@mikebrow"
+owning-sig: sig-node
+status: implementable
+creation-date: 2020-09-30
+reviewers:
+  - "@derekwaynecarr"
+  - "@dchen1107"
+approvers:
+  - "@derekwaynecarr"
+  - "@dchen1107"
+prr-approvers:
+  - "@deads2k"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.5"
```

Comment:
Set to 1.5 per https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/